yew/src/sgml_tags.rs

```diff
@@ -0,0 +1,230 @@
+//! This module contains utilities for parsing or validating strings relating
+//! to tags.
+
+use lazy_static::lazy_static;
+use std::collections::HashSet;
+use std::iter::FromIterator;
+
+lazy_static! {
+    static ref DISALLOWED_CUSTOM_ELEMENT_TAGS: HashSet<&'static str> = HashSet::from_iter(
+        vec![
+            "annotation-xml",
+            "color-profile",
+            "font-face",
+            "font-face-src",
+            "font-face-uri",
+            "font-face-format",
+            "font-face-name",
+            "missing-glyph",
+        ]
+        .iter()
+        .map(|tag| tag.clone())
+    );
```

Comment:
No need to `iter` and `clone` here.
```suggestion
    static ref DISALLOWED_CUSTOM_ELEMENT_TAGS: HashSet<&'static str> = HashSet::from_iter(vec![
        "annotation-xml",
        "color-profile",
        "font-face",
        "font-face-src",
        "font-face-uri",
        "font-face-format",
        "font-face-name",
        "missing-glyph",
    ]);
```

Reply:
Please note that if you agree with my comment further down then this list will no longer be necessary.
At that point we can remove the `lazy_static` dependency.